src/bootloader/bootloaderlite.h

```diff
@@ -35,13 +35,16 @@ class BootFwUpdateStatus {
 class BootloaderLite : public Bootloader, public BootFwUpdateStatus {
  public:
   static constexpr const char* const VersionFile{"/usr/lib/firmware/version.txt"};
+  static constexpr const char* const OstreeTargetPathVar{"FIO_OSTREE_TARGET_SYSROOT"};
 
   explicit BootloaderLite(BootloaderConfig config, INvStorage& storage, OSTree::Sysroot::Ptr sysroot);
 
   VersionStrRes getDeploymentVersion(const std::string& hash) const;
 
   static std::string getVersion(const std::string& deployment_dir, const std::string& hash,
                                 const std::string& ver_file = VersionFile);
+  static std::string getTargetDir(const std::string& deployment_dir,
```

Comment:
I suppose this method can be moved to the `private` section since it's used only within class methods.

Reply:
fixed, thanks